### ISSUE TITLE
"bower install --allow-root" install fails with " EINVAL: invalid argument, rename"

### ISSUE THREAD

**Issue by GordonSmith** [Microsoft/BashOnWindows#152]
```
bower install --allow-root
bower invalid-meta  The "main" field cannot contain minified files
bower invalid-meta  The "main" field cannot contain minified files
bower invalid-meta  The "main" field cannot contain minified files
bower invalid-meta  The "main" field cannot contain minified files
bower invalid-meta  The "main" field cannot contain minified files
bower invalid-meta  The "main" field cannot contain minified files
bower invalid-meta  The "main" field cannot contain minified files
bower invalid-meta  The "main" field cannot contain minified files
bower invalid-meta  The "main" field cannot contain minified files
bower invalid-meta  The "main" field cannot contain minified files
bower invalid-meta  The "main" field cannot contain minified files
bower invalid-meta  The "main" field cannot contain minified files
bower invalid-meta  The "main" field cannot contain minified files
bower invalid-meta  The "main" field cannot contain minified files
bower invalid-meta  The "main" field has to contain only 1 file per filetype; found multiple .js files: ["dist/hpcc-viz-common.min.js","dist/hpcc-viz-api.min.js","dist/hpcc-viz-layout.min.js","dist/hpcc-viz-other.min.js","dist/hpcc-viz-form.min.js","dist/hpcc-viz-graph.min.js","dist/hpcc-viz-tree.min.js","dist/hpcc-viz-chart.min.js","dist/hpcc-viz-amchart.min.js","dist/hpcc-viz-c3chart.min.js","dist/hpcc-viz-google.min.js","dist/hpcc-viz-map.min.js","dist/hpcc-viz-marshaller.min.js"]
bower not-cached    https://raw.githubusercontent.com/cpettitt/dagre/v0.7.3/dist/dagre.js#*
bower resolve       https://raw.githubusercontent.com/cpettitt/dagre/v0.7.3/dist/dagre.js#*
bower not-cached    https://raw.githubusercontent.com/mourner/simpleheat/v0.3.0/simpleheat.js#*
bower resolve       https://raw.githubusercontent.com/mourner/simpleheat/v0.3.0/simpleheat.js#*
bower not-cached    https://raw.githubusercontent.com/d3/d3-plugins/master/hexbin/hexbin.js#*
bower resolve       https://raw.githubusercontent.com/d3/d3-plugins/master/hexbin/hexbin.js#*
bower not-cached    https://github.com/mbostock/topojson-bower.git#~1.6.24
bower resolve       https://github.com/mbostock/topojson-bower.git#~1.6.24
bower not-cached    https://github.com/masayuki0812/c3.git#~0.4.10
bower resolve       https://github.com/masayuki0812/c3.git#~0.4.10
bower not-cached    https://github.com/undashes/colorbrewer.git#~1.0.0
bower resolve       https://github.com/undashes/colorbrewer.git#~1.0.0
bower not-cached    https://github.com/mbostock-bower/d3-bower.git#~3.5.16
bower resolve       https://github.com/mbostock-bower/d3-bower.git#~3.5.16
bower download      https://raw.githubusercontent.com/cpettitt/dagre/v0.7.3/dist/dagre.js
bower download      https://raw.githubusercontent.com/mourner/simpleheat/v0.3.0/simpleheat.js
bower download      https://raw.githubusercontent.com/d3/d3-plugins/master/hexbin/hexbin.js
bower not-cached    https://github.com/jasondavies/d3-cloud.git#~1.2.1
bower resolve       https://github.com/jasondavies/d3-cloud.git#~1.2.1
bower not-cached    https://github.com/Caged/d3-tip.git#~0.6.7
bower resolve       https://github.com/Caged/d3-tip.git#~0.6.7
bower not-cached    https://github.com/components/es6-promise.git#~3.2.1
bower resolve       https://github.com/components/es6-promise.git#~3.2.1
bower checkout      colorbrewer#v1.0.0
bower checkout      topojson#v1.6.24
bower checkout      c3#0.4.10
bower checkout      d3#v3.5.16
bower checkout      d3-cloud#v1.2.1
bower checkout      d3-tip#v0.6.7
bower checkout      es6-promise#v3.2.1
bower EINVAL        EINVAL: invalid argument, rename '/tmp/root/bower/39bbe96029529b9fb2f29cfc9922a1c4-53-fu6oJ3' -> '/root/.cache/bower/packages/0740e5839586c5ab531dded2021d3de1/_wildcard'

Stack trace:
Error: EINVAL: invalid argument, rename '/tmp/root/bower/39bbe96029529b9fb2f29cfc9922a1c4-53-fu6oJ3' -> '/root/.cache/bower/packages/0740e5839586c5ab531dded2021d3de1/_wildcard'
    at Error (native)

Console trace:
Error
    at StandardRenderer.error (/usr/local/lib/node_modules/bower/lib/renderers/StandardRenderer.js:81:37)
    at Logger.<anonymous> (/usr/local/lib/node_modules/bower/lib/bin/bower.js:110:26)
    at emitOne (events.js:90:13)
    at Logger.emit (events.js:182:7)
    at Logger.emit (/usr/local/lib/node_modules/bower/lib/node_modules/bower-logger/lib/Logger.js:29:39)
    at /usr/local/lib/node_modules/bower/lib/commands/index.js:48:20
    at _rejected (/usr/local/lib/node_modules/bower/lib/node_modules/q/q.js:844:24)
    at /usr/local/lib/node_modules/bower/lib/node_modules/q/q.js:870:30
    at Promise.when (/usr/local/lib/node_modules/bower/lib/node_modules/q/q.js:1122:31)
    at Promise.promise.promiseDispatch (/usr/local/lib/node_modules/bower/lib/node_modules/q/q.js:788:41)
System info:
Bower version: 1.7.9
Node version: 5.10.1
OS: Linux 3.4.0+ x64
```

**Comment by hut8** [comment]
I'm going to take a wild guess and say that this may be related to: https://github.com/Microsoft/BashOnWindows/issues/163

Care to try out strace on it?

**Comment by benhillis** [comment]
I tried this in our development branch and I've been able to install packages using bower.  I believe this was fixed with the same symlink fix that resolved #3 and others.

@GordonSmith If you have specific bower packages you'd like me to try out let me know.

**Comment by benhillis** [comment]
I've verified this is fixed in build 14352, please reopen if you continue to see this bug. Thanks!
